### ISSUE TITLE
[Improvement] Gravitinoi UI can create new metalakes but not effectively use them in playground

### ISSUE THREAD

**Issue by justinmclean** [datastrato/gravitino#3246]
### What would you like to be improved?

The Gravitino playground is configured to use simple names but the UI enables you to create new metalakes.

### How should we improve?

Stop UI from being able to create new metalakes when using simple names?

**Comment by shaofengshi** [comment]
User still has the capability to create multiple metalakes. The only change is at the client (trino, spark) side, where they need to specify the metalake name in client configuration.

**Comment by justinmclean** [comment]
The issue is that you can only use one metalake and can't query between them - this can be confusing for the user.

**Comment by shaofengshi** [comment]
> The issue is that you can only use one metalake and can't query between them - this can be confusing for the user.

Hi Justin, cross-metalake query should be very low-freqency: usually they are in the same metalake. But we still keep the possibility: for that case, user can set certain property in the client(Trino) side, after that they can use the full four-section name to do cross-metalake query. @diqiu50 Yu Hui can you confirm this?

**Comment by justinmclean** [comment]
If it is only low frequency then we need to change the UI. Currently, it's the first thing a user will see when they start up the Gravitino UI.
<img width="1466" alt="image" src="https://github.com/datastrato/gravitino/assets/144504/2667f025-7a1d-475f-ad0f-f5441ecd8945">
Note the most prominent call to action is the "Create Metalake" button.

**Comment by justinmclean** [comment]
Their is also no need for the filter on metalakes.

**Comment by diqiu50** [comment]
I think he ui is difference with Trino connector. Users can create a metalake  on ui.  And users can using  one or more metalake in Trino

**Comment by diqiu50** [comment]
<!DOCTYPE html>
gravitino.simplify-catalog-names | boolean | true | The gravitino.simplify-catalog-names setting omits the metalake prefix from catalog names when set to true. | NO | 0.5.0
-- | -- | -- | -- | -- | --

We can set `gravitino.simplify-catalog-names` = false to enable multiple metalakes support

**Comment by justinmclean** [comment]
How do you do that with the playground? By default, it is true, and thus the Gravitino UI can be confusing to a first-time user. In the playground, you can change the config, but no Trino admin tools are installed.

**Comment by diqiu50** [comment]
This requires the Trino admin to modify the Trino connector configuration. Of course, you can also refer to the documentation. [configuring-the-gravitino-connector](https://github.com/datastrato/gravitino/blob/main/docs/trino-connector/installation.md#configuring-the-gravitino-connector)

**Comment by justinmclean** [comment]
Sure, I get the purpose, but again, the UI in the playground makes it very easy for users to break things as the Gravitino UI playground assumes you can have multiple meta lakes, and the Tino playground doesn't.

**Comment by shaofengshi** [comment]
> Sure, I get the purpose, but again, the UI in the playground makes it very easy for users to break things as the Gravitino UI playground assumes you can have multiple meta lakes, and the Tino playground doesn't.

Yeah, the UI has the room to improve. Today it is the first page for user to create metalake, actually in most of cases only 1 metadata is enough. We need to improve it in the future releases, e.g, not put it as the first page.
